app/models/user.rb

```diff
@@ -57,6 +57,7 @@ class User < ApplicationRecord
   has_many :availabilities, dependent: :destroy
   validates :cell, presence: true
   validates :email, presence: true
```

Comment:
Please combine the validations into one line:
`validates :email, presence: true, uniqueness: true`